src/tests/cucumber/features/steps/common_steps/modeler_mps_steps.py

```diff
@@ -0,0 +1,168 @@
+from behave import *
+import xpress as xp
```

Comment:
xpress requires a license and is nonfree, maybe we could find another python package for MPS parsing ?